### PR TITLE
fix icon links and update old screenshot

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

File: desktop/index.md [hunk 1]
```diff
@@ -78,15 +78,15 @@ using a secure repository.
         <div class="component">
              <div class="component-icon">
                  <img src="/assets/images/download.svg" alt="Download and install" width="70" height="70">
```
Comment (dvdksn) [inline desktop/index.md:80]: Looks like the install icon is missing the link

```suggestion
                 <a href="/desktop/install/mac-install"><img src="/assets/images/download.svg" alt="Download and install" width="70" height="70"></a>
```

Comment (aevesdocker) [reply desktop/index.md:80]: Yeh I missed this one out on purpose the descriptions below provide the link to each install page for Mac, Windows, Linux and I wasn't sure which one to attach to the icon

Comment (dvdksn) [reply desktop/index.md:80]: maybe we should remove the link for the h2 element too in that case? it changes the text color but I don't think it looks bad.

image
image

Comment (aevesdocker) [reply desktop/index.md:80]: I'm tempted to leave as is so users can access whichever is appropriate to them

Comment (aevesdocker) [reply desktop/index.md:80]: Fixing now!

**Review by dvdksn — APPROVED** [review]
:shipit:

**Pull request merged**